### PR TITLE
Converted css files to css modules - section 4

### PR DESCRIPTION
**Pull request by sakshikhachane**
## Issue that this pull request solves

 Closes: #349

## Proposed changes
Converted CSS to CSS modules section 4 -

- [x] FAQ.css
- [x] footer.css
- [x] motive.css
- [x] overview.css
- [x] home.css

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @sakshikhachane as we don't have any file named as `motive.css`. Please make the requite changes in [here](https://github.com/HITK-TECH-Community/Community-Website/blob/main/frontend/src/components/home/carousel/carousel.js#L6) and related places along with [here](https://github.com/HITK-TECH-Community/Community-Website/blob/main/frontend/src/components/broadcast/Carousel/Carousel.js#L6) and required places in this file.

**Comment by sakshikhachane** [comment]
@Kajol-Kumari Do I have to make changes in the classes of mentioned files as required for `motive.module.css` or make a new `motive.css` file?

**Comment by sakshikhachane** [comment]
@Kajol-Kumari I have updated the files and done the changes requested. Please have a look.

File: frontend/src/components/home/carousel/carousel.js [hunk 1]
```diff
@@ -38,9 +38,9 @@ export class Owldemo1 extends Component {
   render() {
     return (
       <div className="carousel-div">
-        <div className="motive">
+        <div className={style.motive}>
           <h1 style={{ color: "#121f3a" }}>Curious to know More about Us?</h1>
```
Comment (Kajol-Kumari) [inline frontend/src/components/home/carousel/carousel.js:42]: This is getting distorted for small screen. Can you please check and fix it

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@sakshikhachane Thanks for incorporating the suggested changes. Just a minor change is required. Rest looks good to go :)

**Comment by sakshikhachane** [comment]
@sakshikhachane I have done the changes. Please have a look.

**Comment by Kajol-Kumari** [comment]
@sakshikhachane please resolve the conflicts.

**Comment by sakshikhachane** [comment]
@Kajol-Kumari I have resolved the conflicts and updated the code.

**Review by Kajol-Kumari — APPROVED** [review]

**Pull request merged**